main.go

```diff
@@ -75,14 +75,20 @@ func generate(name string, args []string) {
 		log.Fatalln("error parsing version:", err)
 	}
 
-	addExtRegexp, err := regexp.Compile(*addext)
-	if err != nil {
-		log.Fatalln("error parsing extension inclusion regexp:", err)
+	var addExtRegexp *regexp.Regexp = nil
+	if *addext != "" {
```

Comment:
Would the behavior change if this if statement were `if true {` instead? Or is this strictly an optimization?

I ask because in the documentation flag, there's no mention of special behavior if the flag has empty string value.